pkg/controllers/data.go

```diff
@@ -179,3 +179,19 @@ func getQueryParams(r *http.Request) (query *entities.Query, err error) {
 		FinishDate: finishDate,
 	}, nil
 }
+func (d *DataController) writeResponseByError(w http.ResponseWriter, err error) {
+	switch err.Error() {
+	case "user is not authorized to list thing's data":
```

Comment:
I think it's better to import the referred errors and use them here instead of the raw error string

Reply:
+1

Reply:
Done!